src/plugins/elasticsearch/index.js

```diff
@@ -24,7 +24,7 @@ module.exports = function (kibana) {
           key: Joi.string()
         }).default(),
         apiVersion: Joi.string().default('2.0'),
-        minimumVerison: Joi.string().default('2.0.0')
+        minimumVerison: Joi.string().default('2.1.0')
```

Comment:
Nevermind, @jbudz says this is working despite the typo.  Which means it must be typo'd everywhere.